statistics-dashboard-common/src/main/java/statistics/dashboard/common/view/BreakdownResult.java

```diff
@@ -0,0 +1,36 @@
+package statistics.dashboard.common.view;
+
+import java.util.List;
+
+/**
+ *  Class that represents the filtering results by breakdown.
+ *  It contains value to represent the breakdown and the results of each value
+ */
+
+public class BreakdownResult {
+
+  private String breakdownBy;
+  private List<StatisticsResult> results;
+
+  public BreakdownResult(String breakdownBy, List<StatisticsResult> results) {
+    this.breakdownBy = breakdownBy;
+    this.results = results;
+  }
+
+  public String getBreakdownBy() {
```

Comment:
I think that we should be a bit more diligent here.
1. This is a predefined set of values. So perhaps instead of a String for the `breakdownBy` we could make it an enum?
2. The UI expects these values to be identical to the variable names in `StatisticsFilteringRequest` and `FilteringOptions`. In those two classes, you could use `@JsonProperty()` to force the name to be identical.
3. Then check in swagger whether the name is consistent... you may need to use a swagger annotation on those fields as well.

Reply:
Just created the enum and did the changes